### PR TITLE
Message when user update his info

### PR DESCRIPTION
**Pull request by 2O4**
added message which will appear when a user update his info from "UserUpdateView".

## Description

I'm proposing to have a message appear when a user update his info from the page /users/~update/.
The message is the same from allauth (when a user connect/disconnect/change password etc.).

The `from django.utils.translation import ugettext_lazy as _` is used for internationalization, it specify a translation string.

## Rationale
This make the site UX more harmonious since the user get message when he performs actions on his account he should too when he updates his info.

This could help new devs to understand and learns how to create messages.

## Use case(s) / visualization(s)
[image of the message](https://i.imgur.com/fUXzBgG.png)

### CODE REVIEW

File: {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/views.py [hunk 1]
```diff
@@ -27,6 +29,11 @@ def get_success_url(self):
     def get_object(self):
         return User.objects.get(username=self.request.user.username)
 
+    def form_valid(self, form):
+        form.save()
+        messages.add_message(self.request, messages.INFO, _("Infos succefully updated"))
+        return super(UserUpdateView, self).form_valid(form)
```
Comment (browniebroke) [inline {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/views.py]: We are running Python 3.6+, we don't need to pass the class and instance to `super()`:

```suggestion
        return super().form_valid(form)
```

File: {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/views.py [hunk 2]
```diff
@@ -27,6 +29,11 @@ def get_success_url(self):
     def get_object(self):
         return User.objects.get(username=self.request.user.username)
 
+    def form_valid(self, form):
+        form.save()
+        messages.add_message(self.request, messages.INFO, _("Infos succefully updated"))
```
Comment (browniebroke) [inline {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/views.py]: Typo:

```suggestion
        messages.add_message(self.request, messages.INFO, _("Infos successfully updated"))
```

File: {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/views.py [hunk 3]
```diff
@@ -27,6 +29,11 @@ def get_success_url(self):
     def get_object(self):
         return User.objects.get(username=self.request.user.username)
 
+    def form_valid(self, form):
+        form.save()
```
Comment (browniebroke) [inline {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/views.py]: We don't need to save the form, I think, the [base view is doing it](https://ccbv.co.uk/projects/Django/2.2/django.views.generic.edit/UpdateView/#form_valid) for us when we call `super().form_valid(form)`.

**Comment by 2O4** [comment]
Sorry about the mistakes, all is resolved.

File: {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/views.py [hunk 4]
```diff
@@ -27,6 +29,10 @@ def get_success_url(self):
     def get_object(self):
         return User.objects.get(username=self.request.user.username)
 
+    def form_valid(self, form):
+        messages.add_message(self.request, messages.INFO, _("Infos successfully updated"))
```
Comment (browniebroke) [inline {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/views.py]: Looks good, just need to fix Black formatting:

```suggestion
        messages.add_message(
            self.request, messages.INFO, _("Infos successfully updated")
        )
```

**Comment by browniebroke** [comment]
Thanks :tada:!

**Pull request merged**